docs/src/main/asciidoc/config-reference.adoc

```diff
@@ -147,6 +147,25 @@ quarkus.http.port=9090 <2>
 TIP: It works in the exact same way as Quarkus Application configuration file `application.properties`. Recommendation
 is to use Quarkus `application.properties`.
 
+=== Locations
+
+Additionally, to the default config locationa, Quarkus provides a way to scan additional locations for configuration
+properties files.
+
+The `quarkus.config.locations` configuration property accepts multiple locations separated by a comma `,` and each
+must represent a valid `URI`. The supported `URI` schemes are:
+
+- file or directory (`file:`)
+- classpath resource
+- jar resource (`jar:`)
+- http resource (`http:`)
+
+All loaded sources use the same ordinal of the source that found the `quarkus.config.locations` configuration
+property. For instance, if `quarkus.config.locations` is set as a system property, then all loaded sources have their
+ordinals set to `400` (system properties use `400` as their ordinal). The ordinal may be overridden directly in the
+resource by setting the `config_ordinal` property. Sources are sorted first by their ordinal, then by location order,
```

Comment:
Which `resource`? How do we set `config_ordinal` there?

Reply:
It's a specific thing coming from the MP Config spec. Let me clarify it.